lib/new_relic/agent/supported_versions.rb

```diff
@@ -11,7 +11,7 @@ module Agent
       {
         :type        => :ruby,
         :name        => "MRI",
-        :supported   => ["2.0.0", "~>2.1.0", "~>2.2.0", "~>2.3.0", "~>2.4.0", "~>2.5.0", "~>2.6.0", "~>2.7.0"],
+        :supported   => ["2.0.0", "~>2.1.0", "~>2.2.0", "~>2.3.0", "~>2.4.0", "~>2.5.0", "~>2.6.0", "~>2.7.0", "~>3.0.0"],
```

Comment:
We should remove 2.0 and 2.1, we haven't supported those ruby versions since 6.x. I think it was just not updated then since I didn't even know this file existed.